domains/_discord.okinea.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "okineadev",
+        "email": "81070564+okineadev@users.noreply.github.com"
```

Comment:
Please add another form of contact (like Discord, Twitter, Mastodon, Reddit etc.)

Reply:
Can I add a telegram?

Reply:
> Can I add a telegram?

Yes.

Reply:
Done!